content/cpp/concepts/math-functions/terms/sin/sin.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'sin()'
+Description: 'Returns the sine of an angle (argument) given in radians in the range[-1,1].'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Returns the sine of an angle (argument) given in radians in the range[-1,1].
+
+## Syntax
+
+```cpp
+std::sin(n)
+```
+
+## Example 
+
+The following example showcases the `sin()` function :
+
+```cpp
+#include<iostream>  
+#include<math.h>  
+using namespace std;  
+int main()  
+{  
+  double degree=60;  
+  //convert degree to radian
+  double radian=degree*3.14/180;  
+  cout<<"Sine of an angle is : "<<sin(radian);  
```

Comment:
It would be good to output the angle as well so it reads "Sine of 60 degrees is 0.86576"